packages/sdk/src/transactions/types.ts

```diff
@@ -0,0 +1,166 @@
+import type { Inscription, Ordinal } from "../inscription/types";
+
+// Types are based on bitcoin RPC
+// Refer to https://bitcoincore.org/en/doc/25.0.0/rpc/rawtransactions/decoderawtransaction/
+
+/**
+ * Transaction output (Vout)
+ */
+export type Vout = {
+  /**
+   * Value in BTC
+   */
+  value: number;
+
+  /**
+   * Index
+   */
+  n: number;
+
+  /**
+   * Ordinals
+   */
+  ordinals: Ordinal[];
+
+  /**
+   * Inscriptions
+   */
+  inscriptions: Inscription[];
```

Comment:
@iamcrazycoder could you help link where this is from?

Reply:
Check the RPC method and its response.